GenderPayGap.WebJob/Functions/Functions.SendReminderEmails.cs

```diff
@@ -87,44 +80,25 @@ public void SendReminderEmails([TimerTrigger("0 * * * *")] TimerInfo timer)
                 {
                     try
                     {
-                        SendReminderEmail(user.EmailAddress, sectorType, organisationsOfSectorType);
-                        SaveReminderEmailRecord(user, sectorType);
+                        SendReminderEmail(user, sectorType, organisationsOfSectorType);
                     }
                     catch (Exception ex)
                     {
                         CustomLogger.Error(
                             "Failed whilst sending or saving reminder email.",
                             new
                             {
-                                User = user,
+                                UserId = user.UserId,
                                 SectorType = sectorType,
-                                Organisations = inScopeOrganisationsThatStillNeedToReport,
+                                OrganisationIds = inScopeOrganisationsThatStillNeedToReport.Select(o => o.OrganisationId),
                                 Exception = ex.Message
```

Comment:
These were the lines of code that were causing the function to hang. The CustomLogger couldn't parse `user` or `inScopeOrganisationsThatStillNeedToReport` into JSON

Reply:
Does the `OrganisationIds` property look sensible in the logs?
I'm wondering because it's an `IEnumerable` and wondering if we need to do a `.ToList()` on it.

Reply:
Great thinking and worth checking.

It looks fine 

`Failed whilst sending or saving reminder email.. Log: {"UserId": 13, "SectorType": "Public", "OrganisationIds": [12, 14], "Exception": "Status code 400.....`